.github/workflows/build.yaml

```
@@ -610,60 +670,41 @@ jobs:
-  #   needs: [setup, build, sign-and-package-windows, sign-and-package-mac]
-  #   runs-on: ubuntu-latest
-  #   if: needs.setup.outputs.release_run
-  #   steps:
-  #     # - uses: actions/download-artifact@v8
-  #     #   with:
-  #     #     pattern: "*-installer"
-
-  #     - uses: actions/download-artifact@v8
-  #       with:
-  #         pattern: "*-metadata"
+  release:
```

actionlint (1.7.12)

[error] job "release" needs job "sign-and-package-mac" which does not exist in this workflow

(job-needs)